id-checklist

```diff
@@ -49,20 +49,25 @@ extension of this lifetime is possible; see Section X below.
 After a document becomes an RFC, it will be marked as replaced by that RFC
 in the I-D repository.
 
-[MSK: This paragraph and the next probably need revision.]
-[BJK: Yes!]
-
-I-Ds are generally in the format of an RFC, although they may be rough drafts
-when the first version is submitted. This format is specified fully in
-"Instructions to RFC Authors" (on the RFC Editor's Web pages [RFCED]). In
-brief, an I-D must be submitted in ASCII text, and should be limited to 72
-characters per line and 58 lines per page, and each page ends with a formfeed
-character. Overstriking to achieve bolding or underlining is not acceptable.
-
-PostScript and/or PDF are acceptable, but only when submitted with a matching
-ASCII version (even if figures must be omitted). PostScript and PDF should be
-formatted for use on 8.5x11 inch paper. If A4 paper is used, an image area
-no greater than 254mm high should be used to avoid printing extra pages when
+I-Ds are generally written with a structure similar to that of an RFC,
+although they may be rough drafts when the first version is submitted. This
+structure is specified fully in "Instructions to RFC Authors" (on the RFC
+Editor's Web pages [RFCED]).  With the advent of the "v3" RFC-authoring
```

Comment:
Assuming those instructions are supposed to be https://www.rfc-editor.org/old/instructions2authors.txt they are kind of dated.